### ISSUE TITLE
Invisible (private?) ascents lead to confusing number of ascents count

### ISSUE THREAD

**Issue by killakalle** [theCrag/website#4126]
Apparently, there are invisible (private?) ascents, that are only partially shown to users.

1. Go to https://www.thecrag.com/es/escalar/spain/jerica/route/4850302881
2. Click on the number of ascents (currently "2")
3. See an empty result list

![image](https://user-images.githubusercontent.com/30682796/198837786-ac906444-ff57-4cf8-b867-ac05ecb14d07.png)

This behavior is somewhat confusing for the user, since there is no information what is going on.
From the user perspective this looks wrong.

**Comment by lordyavin** [comment]
I propose to have a note that mentions the hidden ascents.

**Comment by oliverstory** [comment]
Yes even listing the ascents as ‘ascent by private user’ would preserve privacy.

**Comment by quaestor** [comment]
I’ve requested something along these lines 6 years ago, see https://github.com/theCrag/website/issues/1106#issuecomment-223081492

**Comment by lordyavin** [comment]
The question is if we should close this in favor of the old issue.
